packages/exchange-analytics/test/unit/models.test.ts

```diff
@@ -0,0 +1,78 @@
+import { Provider } from '../../src/api/constants'
+import IFundingRate from '../../src/models/funding-rate'
```

Comment:
I made the same mistake on other endpoints and already fixed it. What you are exporting is the model module, therefore call it `FundingRate` and not `IFundinRage` (which is the interface)`.